src/main/java/graphql/validation/Validator.java

```diff
@@ -4,32 +4,7 @@
 import graphql.Internal;
 import graphql.language.Document;
 import graphql.schema.GraphQLSchema;
-import graphql.validation.rules.ArgumentsOfCorrectType;
-import graphql.validation.rules.DeferredDirectiveOnNonNullableField;
-import graphql.validation.rules.DeferredDirectiveOnQueryOperation;
-import graphql.validation.rules.DeferredMustBeOnAllFields;
-import graphql.validation.rules.ExecutableDefinitions;
-import graphql.validation.rules.FieldsOnCorrectType;
-import graphql.validation.rules.FragmentsOnCompositeType;
-import graphql.validation.rules.KnownArgumentNames;
-import graphql.validation.rules.KnownDirectives;
-import graphql.validation.rules.KnownFragmentNames;
-import graphql.validation.rules.KnownTypeNames;
-import graphql.validation.rules.LoneAnonymousOperation;
-import graphql.validation.rules.NoFragmentCycles;
-import graphql.validation.rules.NoUndefinedVariables;
-import graphql.validation.rules.NoUnusedFragments;
-import graphql.validation.rules.NoUnusedVariables;
-import graphql.validation.rules.OverlappingFieldsCanBeMerged;
-import graphql.validation.rules.PossibleFragmentSpreads;
-import graphql.validation.rules.ProvidedNonNullArguments;
-import graphql.validation.rules.ScalarLeafs;
-import graphql.validation.rules.UniqueDirectiveNamesPerLocation;
-import graphql.validation.rules.UniqueFragmentNames;
-import graphql.validation.rules.UniqueOperationNames;
-import graphql.validation.rules.VariableDefaultValuesOfCorrectType;
-import graphql.validation.rules.VariableTypesMatchRule;
-import graphql.validation.rules.VariablesAreInputTypes;
+import graphql.validation.rules.*;
```

Comment:
Please change this back.  We use non .* imports

Reply:
done